src/hooks/useContactFormFCU.ts

```diff
@@ -119,13 +120,13 @@ const useContactFormFCU = () => {
   );
 
   const handleOnSubmitContact = useCallback(
-    async (data: Record<string, any> = {}) => {
-      if (data.structure !== 'Tertiaire') {
+    async (data?: AddressDataType) => {
+      if (data && data.structure !== 'Tertiaire') {
         data.company = '';
       }
```

Comment:
This has no sens... If data is not defined, then data.company will fail...

Reply:
I not sure to understand. In this case, `data` is defined. 
(`if (data && ...) data.company = ''`)

Reply:
my bad !